notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb

```diff
@@ -26,120 +19,38 @@
     "\n",
     "### Prerequisities\n",
     "Before you start make sure you have Elasticsearch cluster running. The cluster must have at least one machine learning (ML) node with enough (4GB) memory."
```

Comment:
nitpick: Do we need 4gb memory? Could we get away with a smaller node with this being a smaller model?

Reply:
You are right. The model used here is only 416MB. We can go with 2GB node.